package.json

```diff
@@ -4,6 +4,7 @@
   "description": "Resolve DID documents for did:peer",
   "type": "module",
   "source": "src/resolver.ts",
+  "main": "./src/resolver.ts",
```

Comment:
does `"./lib/resolver.js"` not work here? The `main` field should point to the compiled entry javascript file